pandas/tests/series/accessors/test_struct_accessor.py

```diff
@@ -0,0 +1,114 @@
+import re
+
+import pytest
+
+from pandas import (
+    ArrowDtype,
+    DataFrame,
+    Index,
+    Series,
+)
+import pandas._testing as tm
+
+pa = pytest.importorskip("pyarrow")
+
+
+class TestStructAccessor:
+    def test_struct_accessor_dtypes(self):
```

Comment:
Could you also test nested struct data e.g struct with a field that is a struct?